srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -81,8 +81,12 @@ defaults
 {%- if pillar['cluster'][grains['id']]['network']['data_nw']['pvt_ip_addr'] %}    
 {% set pvt_ip = pillar['cluster'][grains['id']]['network']['data_nw']['pvt_ip_addr'] %}
 {%- else %}
+{%- if pillar['cluster'][node]['network']['data_nw']['iface']|length > 1 %}
```

Comment:
variable node is undefined in this file. Please use grains['id'] or define node variable first.